_config.yml

```diff
@@ -45,6 +45,7 @@ navbar-links:
     - News Archive: "archive"
   Get involved:
     - Join US-RSE: "join"
+    - Community driven initiatives: "get-involved"
```

Comment:
Hmm I see why you didn't choose a redundant "Get Involved" - I wonder if there is a more concise header for this link? How to participate? Contribute? How to contribute? Ways to contribute? Getting started? Initiatives?

Reply:
How about "Contribute"? I'm open to other suggestions though. I don't have strong feelings either way at this point.

Reply:
I'm good with contribute, I think it's fairly straight forward about what they find at the link, which is already under "Get Involved"